adhocteam/Head-Start-TTADP
# TTA overview widget

## Description of change
This PR builds on #300 adding support for a TTA overview widget on the landing page.


## How to test
- pull changes or run on sandbox
- create activity reports, both approved and not approved
- login with a user that has permissions to multiple regions
- (simulating logging with a different user can be done by running locally, stopping the server, changing the `CURRENT_USER_ID=<number>` in `.env` to a different number and then starting the server)
- note that having access to multiple regions allows to see the region select dropdown
- use the dropdown to change the region
- note that the overview widget, alerts and reports should adjust
- switch back to the previous region
- apply filters 
- switch back and forth between regions
- verify that the filters stay applied

Note * - outstanding issue - currently the dropdown menu won't close on blur

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-143


## Checklists

### Every PR


- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy


### After merge/deploy
